### PR TITLE
CLI: new licences command

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

- New `adf-cli licenses` command (mark "adf-license" as deprecated)

```bash
Usage: adf-cli licenses [options]

Generate a licences report

Options:
  -p, --package <path>  Path to package file (default: package.json in working directory)
  -d, --outDir <dir>    Ouput directory (default: working directory)
  -h, --help            output usage information
```

Examples:

``` basj
# generate report and put it to the current folder
adf-cli licenses

# generate report and put it to the "docs" folder
adf-cli licenses -d ./docs

# generate report based on the custom package.json file
adf-cli licenses -p ./path/to/package.json
```

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/cli/bin/doc/licenseList.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ var spdxCodes = require('spdx-license-list');
 var ejs = require('ejs');
 var program = require('commander');
```
Comment (eromano) [inline lib/cli/bin/doc/licenseList.js:10]: can we just remvoe it?

Comment (DenysVuika) [reply lib/cli/bin/doc/licenseList.js:10]: yes, we can remove, was not sure which scripts are using it

Comment (DenysVuika) [reply lib/cli/bin/doc/licenseList.js]: ok, removed old command

Comment (eromano) [reply lib/cli/bin/doc/licenseList.js]: we can then check it

**Pull request merged**